docs/design/mockups/phase-4-homepage/round-r1-5c-hero-iterations-comparisons.html

```
@@ -0,0 +1,858 @@
+              </div>
+            </div>
+          </div>
+
+          <!-- EV.3 -->
+          <div class="option">
+            <header>
+              <div class="key">EV.3 · Skip venue on hero</div>
+              <div class="name">EventStrip rendered below the hero card</div>
+              <div class="note">
+                Hero stays minimal (kicker · H1 · lead · byline). A
+                compressed `<EventStrip>` component renders below the
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </eventstrip> ], start tag match failed [ <EventStrip> ] on line 665.

(tag-pair)